XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -110,8 +114,67 @@ private void InitializeData()
 
             SampleCodeLayout2.Value = @"<common:ActivityFeedLayout x:Key=""MyFeedLayout"" ColumnSpacing=""12""
                           RowSpacing=""12"" MinItemSize=""80, 108""/>";
+
+
+            // Initialize list of colors for animated scrolling sample
+            IList<string> colors = new List<String>();
+            colors.Add("BlueViolet");
+            colors.Add("BurlyWood");
+            colors.Add("Crimson");
+            colors.Add("DarkCyan");
+            colors.Add("DarkSalmon");
+            colors.Add("MediumAquamarine");
+            colors.Add("DodgerBlue");
+            colors.Add("Firebrick");
+            colors.Add("Goldenrod");
+            colors.Add("Orange");
+            colors.Add("PaleVioletRed");
+            colors.Add("Olive");
+            colors.Add("MediumOrchid");
+            colors.Add("SeaGreen");
+            colors.Add("SandyBrown");
+            colors.Add("SteelBlue");
```

Comment:
var colors = Enum.GetValues(typeof(Colors)).Cast<Color>().Select(c => c.ToString());

Reply:
This exactly didnt work, but I changed it in an upcoming commit to getting all of the colors into a list in a similar way.